src/main/java/io/github/cottonmc/energy/api/ElectricalEnergyType.java

```diff
@@ -1,20 +1,35 @@
 package io.github.cottonmc.energy.api;
 
+import net.minecraft.text.TextComponent;
+import net.minecraft.text.TranslatableTextComponent;
+
 public class ElectricalEnergyType implements EnergyType {
 
-    private final int size;
+    private final int lvPerPacket;
 
-    public ElectricalEnergyType(int maxSize) {
-        this.size = maxSize;
+    public ElectricalEnergyType(int lvPerPacket) {
+        this.lvPerPacket = lvPerPacket;
     }
 
     public int getPacketSize() {
-        return size;
+        return 1;
     }
 
-
     @Override
     public int getMaximumPacketSize() {
-        return this.size;
+        return 1;
```

Comment:
This looks like it'd make it so each electrical energy packet can only deliver 1 of its voltage at a time. I dunno if that's really a good idea.

Reply:
This is something others have felt strongly about; that WU shouldn't be splittable. Hopefully we'll get them to chime in here soon.

Reply:
Ah, I thought we'd been leaning towards the other side. I think your comment is s/WU/packets btw? WU is the base unit, so 1 LV packet = 1 WU, one UHV currently = 1000 WU.

Reply:
Thinking about it, I think the better thing to do here would be, instead of using a multiplier, set both the min and max to the voltage.

Reply:
Do we want to have it not splittable?